buildSrc/src/main/kotlin/com/saveourtool/processbuilder/Publishing.kt

```diff
@@ -0,0 +1,163 @@
+package com.saveourtool.processbuilder
+
+import org.gradle.api.Named
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.api.tasks.bundling.Jar
+import org.gradle.configurationcache.extensions.serviceOf
+import org.gradle.internal.logging.text.StyledTextOutput
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+import org.gradle.kotlin.dsl.*
+import org.gradle.plugins.signing.SigningExtension
+
+fun Project.configurePublishing() {
+    configureGitHubPublishing()
+    configurePublications()
+    configureSigning()
+}
+
+private fun Project.configureGitHubPublishing() {
+    apply(plugin = "maven-publish")
+    configure<PublishingExtension> {
+        println("${rootProject.name}/${project.name}")
```

Comment:
println is left here and are you sure about ` url = uri("https://maven.pkg.github.com/saveourtool/${rootProject.name}/${project.name}")`?